internal/providers/pke/pkeworkflow/delete_vpc_activity.go

```diff
@@ -72,7 +73,7 @@ func (a *DeleteVPCActivity) Execute(ctx context.Context, input DeleteVPCActivity
 	}
 
 	err = cfClient.WaitUntilStackDeleteCompleteWithContext(ctx, &cloudformation.DescribeStacksInput{StackName: &stackName})
-	if err != nil {
+	if err = amazon.NewAwsStackFailure(err, stackName, cfClient); err != nil {
```

Comment:
do we want to collect errors even if the deletion was complete? (the err value from line 75 is not checked)

Reply:
The `err` is checked in the if statement. See the `; err != nil`. If the original `err` is `nil` then `NewAwsStackFailure` returns `nil`